### PR TITLE
[BHX-2021]Add Birmingham logos and amend pages

### PR DESCRIPTION
**Pull request by Waterdrips**
Signed-off-by: Alistair Hey <alistair@heyal.co.uk>

*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

We are adding some logos, changing the 2021 bham pages and adding venue & dates as we have a signed agreement. :tada: :heart: :confetti_ball:

### CODE REVIEW

File: data/events/2021-birmingham.yml [hunk 1]
```diff
@@ -20,24 +22,22 @@ cfp_date_announce: # inform proposers of status
...
-# location_address: "Venue Name, Venue Street, Birmingham" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+location_address: "Millennium Point, Curzon St, Birmingham B4 7XG" # Defaults to city, but you can make it the venue name.
 
 nav_elements: # List of pages you want to show up in the navigation of your page.
   # - name: propose
   # - name: location
-  # - name: registration
-  #   url: https://ti.to/devopsdays-birmingham/dodb-2021
+  #  - name: registration
+  #    url: https://ti.to/devopsdays-birmingham/dodb-2021
```
Comment (bridgetkromhout) [inline data/events/2021-birmingham.yml]: Your registration link is still commented out, in case that was not intentional.

**Comment by bridgetkromhout** [comment]
Hi, we prefer that you don't leave PRs in Draft - if you aren't going to be able to finish it in the next day, we'll close it and you can re-submit when you have time. Thanks!

File: data/events/2021-birmingham.yml [hunk 2]
```diff
@@ -20,24 +22,22 @@ cfp_date_announce: # inform proposers of status
...
-#
-# location_address: "Venue Name, Venue Street, Birmingham" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+location_address: "Millennium Point, Curzon St, Birmingham B4 7XG" # Defaults to city, but you can make it the venue name.
 
 nav_elements: # List of pages you want to show up in the navigation of your page.
   # - name: propose
   # - name: location
-  # - name: registration
-  #   url: https://ti.to/devopsdays-birmingham/dodb-2021
+  #  - name: registration
```
Comment (TomTucka) [inline data/events/2021-birmingham.yml]:
```suggestion
  - name: registration
```

File: data/events/2021-birmingham.yml (hunk 1 above)
Comment (TomTucka) [inline data/events/2021-birmingham.yml]:
```suggestion
      url: https://ti.to/devopsdays-birmingham/dodb-2021
```

Comment (Waterdrips) [reply data/events/2021-birmingham.yml]: Thanks, un-commented

**Comment by Waterdrips** [comment]
Hi @bridgetkromhout - ready for review. Thanks

**Pull request merged**